apps/src/music/types.ts

```diff
@@ -19,6 +19,7 @@ export interface MusicLevelData extends ProjectLevelData {
   showAiGenerateAgainHelp?: boolean;
   allowChangeStartingPlayheadPosition?: boolean;
   toolboxDefinition?: GoogleBlockly.utils.toolbox.ToolboxInfo;
+  exemplarSources?: Source;
```

Comment:
nit: unused? Looks like we pull exemplar sources from the existing `levelProperties` field right?

Reply:
Oh good catch! I had initially envisioned that is being a `levelData.` property and missed cleaning that back up. :)